automation/doc.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022-2023 Searis AS
+// Copyright 2022-2024 Searis AS
```

Comment:
You only need to update the copy-right header for files you have changed; nice if you can revert this change for files that are otherwise unchanged in the commit.

Reply:
i did this due to a linter error i encountered, so i figured i'd fix all the discrepancies